src/OpenLoco/Message.cpp

```diff
@@ -0,0 +1,165 @@
+#include "Message.h"
+#include "Audio/Audio.h"
+
+namespace OpenLoco
+{
+    // 0x004F8BE4
+    static constexpr uint16_t _MessageTypeFlagsArr[31] = {
```

Comment:
Could we introduce a constant for this `31`? I've seen it pop up in several places.

Reply:
when we know all of the message types we can have one added.

Reply:
Oh, I was under the impression we had already uncovered all types, given the four lookup tables.